### DISCUSSION TITLE
Should we release 0.72.11?

### DISCUSSION THREAD

**Discussion by huntie** [reactwg/react-native-releases#100] [Patches]
# Should we release 0.72.11?

Current Release: [0.72.10](https://github.com/facebook/react-native/releases/tag/v0.72.10)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

All qualified picks have been merged.

We are no longer accepting requests and will be testing and shipping 0.72.12 next week (02/19/2024)

**Comment by kirillzyusko** [comment]
Would be good to include https://github.com/facebook/react-native/pull/42075 👀

It wasn't a problem to use `install_modules_dependencies` in RN 0.71 and in RN 0.73, but 0.72 can not use `install_modules_dependencies` (if library is using Swift code) and PR from above solves the problem (I hope it can be treated as qualified pick).

**Reply by lunaleaps** [reply]
✅ merged

**Comment by liamjones** [comment]
@huntie the current release link in the OP is for the wrong version

**Comment by matinzd** [comment]
Please pick https://github.com/facebook/react-native/commit/96ed1190c624869af96e464b009e0c0234076893

This fixes the issue regarding the permission contracts on Android 14. Currently it’s blocking https://github.com/matinzd/react-native-health-connect/pull/60 and anything related to this matter.

**Reply by lunaleaps** [reply]
✅ picked

**Comment by kagod-dev** [comment]
Would be awesome to include [facebook/react-native@b9418315e27](https://github.com/facebook/react-native/commit/b9418315e274f8693d1f7cd3f41529ba6c0436c9).

It fixes the crash in `getChildDrawingOrder` on Android - [facebook/react-native#40859](https://github.com/facebook/react-native/pull/40859)

**Reply by lunaleaps** [reply]
✅ picked

**Reply by kagod-dev** [reply]
@lunaleaps when it will be released? I see 0.72.10 as the latest

**Reply by kirillzyusko** [reply]
@kagod-dev it's already on npm - you can check a version there 👀

**Comment by robhogan** [comment]
CLI and Metro 0.76.9 update to fix [quite a few](https://github.com/facebook/metro/releases/tag/v0.76.9) Metro issues:

https://github.com/facebook/react-native/pull/42872

**Reply by lunaleaps** [reply]
✅ this has been merged

**Comment by ken0nek** [comment]
Hi,

I created a pull request to fix the `EXCLUDED_ARCHS` override problem on installing hermes. [The same fix is done on RN v0.71](https://github.com/facebook/react-native/pull/39763) but not on RN 0.72 yet.

https://github.com/facebook/react-native/pull/43002

**Reply by lunaleaps** [reply]
✅ merged
